### ISSUE TITLE
Can CalDavSynchronizer designate calendars as read-only

### ISSUE THREAD

**Issue by jmfoleyjr** [aluxnimm/outlookcaldavsynchronizer#259]
I have configured CalDavSynchronizer on Window 7 Outlook 2010, and Windows 10 Outlook 365 to access several CalDAV calendars. The issue is that some of those calendars are read-only on the server end. That is, users can read, but not write to the calendar. On the Outlook end, there is no ability to mark a calendar as read-only. So, if a user happens to try and add an event to a CalDAV calendar to which she has no access, it does put the event on the local screen, but generates a CalDavSynchronizer "Synchronization Report" error report.

Is there a way to mark a calendar as read-only so this error can be avoided?

**Comment by aluxnimm** [comment]
Outlook folders via IMAP or lokal pst files cant be marked read only. But you should just change the sync direction to Outlook <- Server one way then there won't be any sync errors anyway for readonly resources.

**Comment by jmfoleyjr** [comment]
Thanks - I'll try that now. What is the difference between "Outlook <- Server (Replicate)" and "Outlook <- Server (Merge)"? I don't get it just based on the terms "Replicate" and "Merge". Does "Replicate" clobber any entries made on the local calendar which are not on the Server calendar? If so, that's what I want.

**Comment by jmfoleyjr** [comment]
OK, I've set the calendars I want to be "read-only" to "Outlook <- Server (Replicate)" (assuming that means clobbering any entries on the local calendar which as not on the Server calendar, as mentioned in my previous post). That does seem to have worked to prevent the big, scarey error windows from popping up. Interestingly, I had to actually delete the existing calendars from Outlook and re-add them or the errors kept popping up.
